<launch>
	<node  pkg="objects_of_daily_use_finder" type="odu_finder" name="odu_finder" output="screen" >
		<param  name="command" type="string" value="/load" />

		<param  name="database_location" type="string" value="$(find objects_of_daily_use_finder)/database/" />
		<param  name="images_for_visualization_directory" type="string" value="$(find objects_of_daily_use_finder)/data/" />

		<param  name="image_topic" type="string" value="/narrow_stereo/left/image_rect" />

		<param  name="enable_clustering" type="int" value="1" />
		<param  name="enable_incremental_learning" type="int" value="0" />
		<param  name="visualization" type="string" value="/true" />
		
		<param  name="votes_count" type="int" value="10" /> 

		<param  name="tree_k" type="int" value="6" /> 
		<param  name="tree_levels" type="int" value="6" /> 

		<param  name="unknown_object_threshold" type="double" value="0.9" />

		<param  name="radius_adaptation_r_min" type="double" value="200" />
		<param  name="radius_adaptation_r_mAX" type="double" value="600" />
		<param  name="radius_adaptation_K" type="double" value="0.02" />
		<param  name="radius_adaptation_A" type="double" value="800" />
	</node>
</launch>
